shoulder/transform/remove_redundant_am.py

```diff
@@ -0,0 +1,36 @@
+#
+# Shoulder
+# Copyright (C) 2018 Assured Information Security, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from shoulder.transform.abstract_transform import AbstractTransform
+
+
+class RemoveRedundantAccessMechanisms(AbstractTransform):
+    @property
+    def description(self):
+        d = "removing redundant access mechanisms"
+        return d
+
+    def do_transform(self, reg):
+        for key, am_list in reg.access_mechanisms.items():
+            reg.access_mechanisms[key] = am_list[1:]
```

Comment:
I think this should be `reg.access_mechanisms[key] = am_list[:1]`  or `... am_list[0]` instead, right? If we are trying to remove all but one of the mechanisms of the same type, we should just select the first one? Right now, this is just omitting the first one and keeping the rest of the list.

Reply:
Haha, thats a pretty bad screw up on my part... Thanks, I'll fix it to ```am_list[0]``` (thats what I was intending)

Reply:
I went ahead and did a force push to fix this issue right in the commit it was a part of, since it was such a small change. 

If your happy with the patch series, it should be good to merge! Otherwise, open some new comments on the latest forced update.